src/chrome/content/rules/ebay.fr.xml

```diff
@@ -0,0 +1,242 @@
+<!--
+
+	For other eBay coverage, see ebay.com.xml
+	
+	
+	Non-functional subdomain:
+	
+		- ebay.fr	(h)
```

Comment:
Please add this host and redirect to secured www.
This also affects other PRs in #13603.

Reply:
Added redirect